MathLib/LinAlg/LinAlg.h

```diff
@@ -206,6 +210,16 @@ namespace LinAlg
 
 // Vector
 
+/**
+    Set local accessible vector in order to get entries. Call this
+    before call operator[] or get(...) of x.
+    The function only has computation if DDC is appied,
+    e.g. parallel computing. Up to now, Eigen vector is not used for
+    global vectors in parallel computing. Therefore this function is
+    empty in the current status.
+*/
```

Comment:
I would like to  leave it like it is because we may want another parallel approach without using PETSc solver. In the case of such parallel computing case without PETSc solver and using Eigen vector type, this function can be filled.